src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -398,6 +404,13 @@ class ApplicationManagerImpl
   void set_driver_distraction_state(
       const hmi_apis::Common_DriverDistractionState::eType state) OVERRIDE;
 
+  /**
+   * @brief Sets state for lock screen dismissal state (enabled/disabled).
+   * @param lock_screen_dismissal_enabled - New state to be set.
+   */
+  virtual void set_lock_screen_dismissal_state(
```

Comment:
@IGapchuk virtual is redundant as OVERRIDE can be applied to virtual methods only

Reply:
@AByzhynar Fixed 677f681b502cb4f2b5883d15b1da64492d4e33e1.